pkg/themasys/bind.go

```diff
@@ -1,9 +1,11 @@
-package kindsys
```

Comment:
[Self-reminder]
This was consumed externally, for instance from [here](https://github.com/grafana/grafana/blob/main/.github/workflows/scripts/kinds/verify-kinds.go#L25), so we'll need to update these references once we get this merged, otherwise everything will crash.

cc/ @AgnesToulet